### PR TITLE
[jira] Do not fail when `fzf` is not available, make warning clearly visible

### PR DESCRIPTION
**Pull request by artms**
Looks something like this:
image

### CODE REVIEW

File: src/uber/UberTask.php [hunk 1]
```diff
@@ -123,9 +123,23 @@ function ($v1, $v2) {
...
-      $fzf = id(new UberFZF())
-        ->requireFZF()
-        ->setMulti(50)
+      $fzf = id(new UberFZF());
+      if (!$fzf->isFZFAvailable()) {
+        $this->console->writeOut(
+          "<bg:red>** %s **</bg>\n<bg:red>** %s **</bg>\n"
+          "<bg:red>** %s **</bg> %s\n"
+          "<bg:red>** %s **</bg>\n<bg:red>** %s **</bg>\n",
+          pht('WARNING'),
```
Comment (Pawka) [inline src/uber/UberTask.php:132]: Remove those multiple empty WARNING lines. :)

Comment (artms) [reply src/uber/UberTask.php:132]: It is on purpose to make WARNING more visible...

File: src/uber/UberTask.php [hunk 2]
```diff
@@ -123,9 +123,23 @@ function ($v1, $v2) {
...
       // prompt user to choose from menu
-      $fzf = id(new UberFZF())
-        ->requireFZF()
-        ->setMulti(50)
+      $fzf = id(new UberFZF());
+      if (!$fzf->isFZFAvailable()) {
+        $this->console->writeOut(
+          "<bg:red>** %s **</bg>\n<bg:red>** %s **</bg>\n"
+          "<bg:red>** %s **</bg> %s\n"
+          "<bg:red>** %s **</bg>\n<bg:red>** %s **</bg>\n",
```
Comment (artms) [inline src/uber/UberTask.php:131]:
```suggestion
          "<bg:red>** %s **</bg>\n<bg:red>** %s **</bg>\n".
          "<bg:red>** %s **</bg> %s\n".
          "<bg:red>** %s **</bg>\n<bg:red>** %s **</bg>\n",
```

**Review by Pawka — APPROVED** [review]

File: src/uber/UberTask.php (hunk 1 above)
Comment (Pawka) [reply src/uber/UberTask.php:132]: The warning message is inconsistent with whole Arcanist flow. Also it translates the message this particular warning is more important compared to other warnings.

**Pull request merged**